packages/components/src/Radio/Radio/_docs/Radio.stickersheet.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { Radio } from "../index"
+
+export default {
+  title: "KAIO-staging/Radio controls/Radio",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+    a11y: {
+      config: {
+        rules: [
+          {
+            // Built with no label on purpose, to be used within `RadioField` where label is present
+            id: "label",
+            enabled: false,
+          },
+        ],
+      },
+    },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StickerSheetStory = {
+  render: ({ isReversed }) => (
+    <StickerSheet isReversed={isReversed}>
+      <StickerSheet.Body>
+        <StickerSheet.Row>
+          <Radio id="radio" name="radio" value="radio" reversed={isReversed} />
+        </StickerSheet.Row>
+        <StickerSheet.Row>
+          <Radio
+            id="radio"
+            name="radio"
+            value="radio"
+            reversed={isReversed}
+            selectedStatus
+          />
```

Comment:
We should also add the pseudo stories in the [new format](https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3230040105/2.3+Testing#Testing-pseudo-states) Cass has whipped up, which should look like this:

```jsx
  render: () => (
    <Radio
      {...radioProps}
      data-sb-pseudo-styles="hover"
    />
  ),
  parameters: {
    pseudo: {
      hover: '[data-sb-pseudo-styles="hover"]',
   }
```

Reply:
Unfortunately doesn't work cause neither the `classNameOverride` or the `data-attr` reach the actual child element that triggers the hover state, but focus works!